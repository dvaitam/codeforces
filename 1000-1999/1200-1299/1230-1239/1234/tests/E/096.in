19 3
2 5 6
