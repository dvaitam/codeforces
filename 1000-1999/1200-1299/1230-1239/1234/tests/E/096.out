4 5 3 3 7 8 4 4 4 4 4 4 4 4 4 4 4 4 4
